File: C_Sharp_code/Example/WindowsFormsApp/CS_demo21/WidescreenTV.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace CS_demo21
{
    public class WidescreenTV : Television
    {
        public override void decreaseVolume()
        {
            Console.WriteLine("volume decrease(WidescreenTV)");
        }

        public override void increaseVolume()
        {
            Console.WriteLine("volume increase(WidescreenTV)");
        }

        public override string ToString()
        {
            return "This's Wide Screen TV";
        }
    }
}

File: C_Sharp_code/Example/WindowsFormsApp/VB_demo22/WidescreenTV.vb
Public Class WidescreenTV
    Inherits Television

    Public Overrides Sub IncreaseVolume()
        Console.WriteLine("increase volume(WidescreenTV)")
    End Sub

    Public Overrides Sub DecreaseVolume()
        Console.WriteLine("decrease volume(WidescreenTV)")
    End Sub

    Public Overrides Function ToString() As String
        Return "This's Wide Screen TV"
    End Function
End Class
